### PR TITLE
[IE Samples] use set_spatial_dynamic_shape in classification async

### PR DESCRIPTION
**Pull request by AlexeyLebedev1**
### Details:
- *Provides example of preprocessing with openvino in case several images*

### CODE REVIEW

**Review by vladimir-dudnik — COMMENTED** [review]
please fix basic pre-commit checks

**Review by dpigasin — COMMENTED** [review]
With these changes the sample doesn't work on my system. It exits without any error after step 5.

File: samples/python/classification_sample_async/classification_sample_async.py [hunk 1]
```diff
@@ -94,9 +90,15 @@ def main() -> int:
     # - input() provides information about a single model input
     # - precision of tensor is supposed to be 'u8'
     # - layout of data is 'NHWC'
+    # - set dynamic spatial dimensions to input tensor to resize from
     ppp.input().tensor() \
         .set_element_type(Type.u8) \
-        .set_layout(Layout('NHWC'))  # noqa: N400
+        .set_layout(Layout('NHWC')) \
+        .set_spatial_dynamic_shape() # noqa: N400
```
Comment (dpigasin) [inline samples/python/classification_sample_async/classification_sample_async.py:97]:
```suggestion
        .set_spatial_dynamic_shape()  # noqa: N400, ECE001
```

Comment (dpigasin) [reply samples/python/classification_sample_async/classification_sample_async.py:97]: linter errors fix

**Review by dpigasin — COMMENTED** [review]
What the model did you use to test with?

**Comment by AlexeyLebedev1** [comment]
> What the model did you use to test with?
I tested it with mobilenet-v1-1.0-224, but this changes don't limit a model set to run this sample.

**Comment by dpigasin** [comment]
> > What the model did you use to test with?
> > I tested it with mobilenet-v1-1.0-224, but this changes don't limit a model set to run this sample.

ok, but I have `RuntimeError: Only dynamic batch is supported!` with the last nightly package. What could it be?

**Comment by dpigasin** [comment]
> > > What the model did you use to test with?
> > > I tested it with mobilenet-v1-1.0-224, but this changes don't limit a model set to run this sample.
> 
> ok, but I have `RuntimeError: Only dynamic batch is supported!` with the last nightly package. What could it be? Old MO?

It relates to GPU, on CPU works fine.

**Comment by AlexeyLebedev1** [comment]
`set_spatial_dynamic_shape` is not supported on GPU, closing this PR.

**Pull request closed without merging**